packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -7,12 +7,15 @@ import { observer, useStore } from '@deriv/stores';
 import './asset-summary.scss';
 import TotalAssetsLoader from 'Components/pre-loader/total-assets-loader';
 import { useTotalAccountBalance, useCFDAccounts, usePlatformAccounts, useExchangeRate } from '@deriv/hooks';
+import { useCashierStore } from '../../../../cashier/src/stores/useCashierStores';
```

Comment:
@shontzu-deriv Please don't import stuff cross packages 🙏🏻

Reply:
@ali-hosseini-deriv Could we add a simple CI check to avoid this? We just simply need to check for changed files if they contain `../[package_names]` and fail the CI.

cc: @yashim-deriv

Reply:
totally. makes sense 

but in that case, how do I implement the fix raised for [this card](https://app.clickup.com/t/20696747/CFDS-1097)...?

since I need the `is_loading` and `is_transfer_confirm` status from the cashier package to implement the fix